app/util/StringUtils.scala

```diff
@@ -79,4 +80,22 @@ object StringUtils {
     */
   def prettifyDateAndTime(date: Date)(implicit messages: Messages): String =
     DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, messages.lang.locale).format(date)
+
+  //https://stackoverflow.com/a/9855338
+  private val hexArray = "0123456789ABCDEF".toCharArray
+  def bytesToHex(bytes: Array[Byte]): String = {
```

Comment:
Can't you do https://stackoverflow.com/a/2756627 here?

Reply:
Holy shit, a Kashike :P
Anyway, from my own guesses, and from the comments seen here, because it's much slower